src/test/java/guitests/BoardTests.java

```diff
@@ -277,15 +279,15 @@ public void boards_panelCount_newBoardCreated() {
         PanelControl panelControl = ui.getPanelControl();
 
         traverseMenu("Boards", "New");
-        waitUntilNodeAppears(hasText("OK"));
-        clickOn("OK");
+        waitUntilNodeAppears(hasText("No"));
+        clickOn("No");
```

Comment:
Should put a comment why we click `No` because it is not obvious. When you want a new board to be created, when we click `New` it seems more intuitive to choose `Yes`, e.g. the prompt is `Your current board is not saved. Open a new board anyway?`.